source/microbit/help.c

```diff
@@ -33,40 +33,27 @@
 STATIC const char *help_text =
 "Welcome to MicroPython on the micro:bit!\n"
 "\n"
-"Type 'import microbit', press return and try these commands:\n"
-"  microbit.display.scroll('Hello')\n"
-"  microbit.running_time()\n"
-"  microbit.sleep(1000)\n"
-"  microbit.button_a.is_pressed()\n"
+"Try these commands:\n"
+"  display.scroll('Hello')\n"
+"  running_time()\n"
+"  sleep(1000)\n"
+"  button_a.is_pressed()\n"
 "What do these commands do? Can you improve them? HINT: use the up and down\n"
 "arrow keys to get your command history. Press the TAB key to auto-complete\n"
-"unfinished words (so 'mi' becomes 'microbit' after you press TAB). These\n"
+"unfinished words (so 'di' becomes 'display' after you press TAB). These\n"
 "tricks save a lot of typing and look cool!\n"
 "\n"
 "Explore:\n"
 "Type 'help(something)' to find out about it. Type 'dir(something)' to see what\n"
```

Comment:
I'd argue that "something" is just fine, otherwise I'd be like explain what "any key" is, users might be young but perfectly capable to understand what this means.